pytradfri/gateway.py

```diff
@@ -50,14 +54,35 @@ def process_result(result):
         )
 
     @classmethod
-    def get_endpoints(cls):
+    def get_endpoints(cls) -> Command:
         """
         Return all available endpoints on the gateway.
 
+        The response from the gateway looks like this:
+        <//15006>;
```

Comment:
The example here will better match how we split the string of we make each line end with the comma.